spec/requests/support_interface/mailer_previews_spec.rb

```diff
@@ -0,0 +1,216 @@
+require 'rails_helper'
+
+RSpec.describe 'Support interface - Mailer previews', type: :request do
```

Comment:
Can we loop through the mailers for these tests, so they automatically are run when added?

Reply:
Hmm, yes, however in that case you'd only be able to `xit` an entire mailer instead of an individual broken preview. I think that's preferable once they're all green, so will update to that.

Reply:
@tijmenb see updated code. I've used `.emails.each` but also had to define a list of `broken_previews` which are currently failing. These are marked as pending. The idea is to fix in follow-up PRs.